index.html

```diff
@@ -25,13 +23,31 @@ <h1 id="" class="">Las paisas</h1>
 			<div class="homepage-hero__divider"></div>
 			<span id="" class="homepage-hero__promotext">El paraiso te espera</span>
 		</article>
-	</section>
+	</section></html-->
 	<section>
 		<!-- comodities section -->
 	</section>
-	<footer>
-		<!-- footer section -->
-	</footer>
+	<footer class="footer--rectangle">
```

Comment:
I'm confused, at the BEM conventions is "footer--rectangle" or "footer__rectangle"

Reply:
for this case it should be only `footer`
as
`footer--rectangle` implies a modifiesd footer, but this is the base footer
and
`footer__rectangle` implies a sub-element of `footer`

Reply:
@jhonatanrojasbastidas Please fix the naming convention.

Reply:
last commit i fix that.

Reply:
> I'm confused, at the BEM conventions is "footer--rectangle" or "footer__rectangle"

yes, i confused there but is fixed.